packet_injector/server.go

```diff
@@ -40,7 +40,23 @@ const (
 
 // PacketInjectorServer creates a packet injector server API
 type PacketInjectorServer struct {
-	Graph *graph.Graph
+	Graph    *graph.Graph
+	Channels *Channels
+}
+
+func (pis *PacketInjectorServer) stopPI(msg *shttp.WSJSONMessage) error {
+	var uuid string
+	if err := common.JSONDecode(bytes.NewBuffer([]byte(*msg.Obj)), &uuid); err != nil {
```

Comment:
something is missing as result of this if ?

Reply:
added a return statement